addons/event/models/event_registration.py

```diff
@@ -22,6 +22,8 @@ class EventRegistration(models.Model):
     event_id = fields.Many2one(
         'event.event', string='Event', required=True,
         readonly=True, states={'draft': [('readonly', False)]})
+    event_name = fields.Char('Event name', related='event_id.name')
+    event_type_id = fields.Many2one('event.type', string='Event Category', related='event_id.event_type_id', store=True)
```

Comment:
Does not seem to be used, is it ?

Reply:
Indeed, it was used for the search panel, but we finally dropped it